config/log/logger_adapter.go

```diff
@@ -0,0 +1,34 @@
+package log
+
+import (
+	"github.com/anchore/go-logger"
```

Comment:
I see that having a separate interface makes the lib more portable, however, we've started to lean away from this and have a common logging interface. Right now since this import is here it removes the value of having a locally defined interface.

Reply:
I don't necessarily agree that having an adapter for known usages removes the value of having a simpler local interface (we could add an interface to the standard go logger here, too, which would make this library even more approachable)